src/Locator.cpp

```diff
@@ -117,6 +118,7 @@ bool openblack::InitializeEngine(uint8_t rendererType, bool vsync) noexcept
 
 bool openblack::InitializeGame() noexcept
 {
+	Locator::textRenderer::emplace<TextRenderer>();
```

Comment:
This needs to be reset at `~Game`. I know it's horrible. But that's how it is right now.